openedx/features/subscriptions/admin.py

```diff
@@ -0,0 +1,23 @@
+"""
+Django Admin pages for Subscriptions.
+"""
+
+from django.contrib import admin
+
+from openedx.features.subscriptions.models import UserSubscription
+
+
+class UserSubscriptionAdmin(admin.ModelAdmin):
+    """
+    Admin interface for the "UserSubscription" object.
+    """
+    list_display = ['username', 'expiration_date', 'number_of_courses', 'subscription_type', 'course_enrollments_slugs']
```

Comment:
add `search_fields = ['user__email', 'user__username']`

Reply:
done

Reply:
Order of fields should be:
`['username', 'subscription_type', 'expiration_date', 'max_allowed_courses', ', 'enrolled_course_ids']`

Also `filter_horizontal` for course enrollments for a better view. (https://stackoverflow.com/questions/5385933/a-better-django-admin-manytomany-field-widget, https://stackoverflow.com/questions/25776235/many-to-many-fields-view-on-django-admin/25776280)

Reply:
Done.